public/editor/scripts/project/project.js

```diff
@@ -237,22 +238,34 @@ define(function(require) {
             return callback(err);
           }
 
-          // Find an HTML file to open in the project, hopefully /index.html
-          var sh = new _fs.Shell();
-          sh.find(getRoot(), {name: "*.html"}, function(err, found) {
-            if(err) {
-              return callback(err);
+          // Find the index.html file in the project root to open
+          var indexLocation = Path.join(getRoot(),"index.html");
+          _fs.exists(indexLocation, function(exist) {
```

Comment:
The variable should be named `exists` not `exist`.  The node.js code for `fs.exists` is deprecated, but we aren't actually using node's code. We wrote a library for the browser that acts like node's `fs` but is different.  You could also use `fs.stat`, but I think it's fine to just do `fs.exists` instead.